From 74db01097a5ce34152bf2d7b049ec2d72423f4e7 Mon Sep 17 00:00:00 2001
From: "Tekriwal, Tanuj" <tanuj.tekriwal@intel.com>
Date: Thu, 2 Jan 2025 07:49:07 +0000
Subject: [PATCH] Updating libva build libs according to latest Code

This patch will update the libva libs based on latest
AOSP and intel dependencies.

Tests Done: Build and boot

Tracked-On: OAM-128386
Signed-off-by: Tekriwal, Tanuj <tanuj.tekriwal@intel.com>
---
 Android.bp | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/Android.bp b/Android.bp
index 39bd5a1..0af3d25 100644
--- a/Android.bp
+++ b/Android.bp
@@ -23,7 +23,7 @@ cc_genrule {
 }
 
 cc_library_shared {
-    name: "libva",
+    name: "libva_orig",
     srcs: [
         "va/va.c",
         "va/va_trace.c",
@@ -86,7 +86,7 @@ cc_library_headers {
 }
 
 cc_library_shared {
-    name: "libva-android",
+    name: "libva-android_orig",
     srcs: [
         "va/android/va_android.cpp",
         "va/drm/va_drm_utils.c",
@@ -102,8 +102,8 @@ cc_library_shared {
         "va/drm",
     ],
     shared_libs: [
-        "libva",
-        "libdrm",
+        "libva_orig",
+        "libdrm_orig",
         "liblog",
     ],
     static_libs: [
-- 
2.34.1

